### ISSUE TITLE
[android] platform view scrolling performance issue

### ISSUE THREAD

**Issue by delfme** [flutter/flutter#119337]
I have found out an issue with scrolling flutter ListView when a native widget is on the screen.
I have always thought this issue was due to overall performance limits with platform views.
But lately I noticed it might be due to gestures (see Steps to Reproduce).

Tested it on master 3.7.0-28.0.pre.7, android 12SKQ1.211006.001

## Steps to Reproduce

Unfortunately code is closed, but you can check vid below. 
On the screen we have a flutter listView with a few flutter text elements. 
On the bottom we also have a simple wrapper to android EditText.

Scrolling listView is janky on android.
If I keep the finger on the screen during scrolling, it is not janky anymore.
This makes me think that issue may be related to gestures somehow.

**Expected results:**
Listview should scroll smoothly. It is such as long as my finger is kept on the screen during scrolling (vid below).

https://user-images.githubusercontent.com/53510751/215118969-87dd799a-3fee-42d5-847a-fc5fac37dbe8.MP4

**Actual results:**
Listview scrolling is janky (here I'm raising up my finger during scrolling)

https://user-images.githubusercontent.com/53510751/215118882-0d344f90-8f61-4c38-9363-8a2e519a77df.MP4

**Flutter doctor:**
 [✓] Flutter (Channel master, 3.7.0-28.0.pre.7, on macOS 12.2 21D49 darwin-arm64, locale en-IT) [!] Android toolchain - develop for Android devices (Android SDK version 31.0.0) ✗ cmdline-tools component is missing Run path/to/sdkmanager --install "cmdline-tools;latest" See https://developer.android.com/studio/command-line for more details. ✗ Android license status unknown. Run flutter doctor --android-licenses to accept the SDK licenses. See https://flutter.dev/docs/get-started/install/macos#android-setup for more details. [✓] Xcode - develop for iOS and macOS (Xcode 13.4.1) [✓] Chrome - develop for the web [✓] Android Studio (version 2021.2) [✓] Connected device (3 available) [✓] HTTP Host Availability

**Comment by darshankawar** [comment]
Thanks for the report @delfme 
Can you please check [this](https://github.com/flutter/flutter/issues/110414) similar issue and see if it resonates with your case ?

**Comment by delfme** [comment]
Hi @darshankawar in this case I don’t see distortion while scrolling and fast scrolling is not required to see issue. 

Basically listView scrolling is janky during a normal scrolling (finger initiates the swipe and lifts off the screen). But if finger remains on the screen during the scrolling, then animation is smooth.

It seems something occurs when engine detects a finger-up event of a swipe gesture, and that makes the ongoing listView animation janky.

**Comment by darshankawar** [comment]
Thanks for the update @delfme 


> On the bottom we also have a simple wrapper to android EditText.

Is this wrapped with GestureDetector` in your code ? Could be that if this is the case, then gesture detector events may not be firing but this is my guess.

A proper reproducible code sample would be helpful to make this issue actionable.

**Comment by delfme** [comment]
Yes but why this should be an issue? 🤔I mean, I dont tap the texifield before or while I scroll the flutter listview and demo works smoothly on ios.

**Comment by darshankawar** [comment]
Could be related to how native Android (read platform views) responds when it is being used as part of the gestures while scrolling, so a reproducible code sample would be helpful.

**Comment by delfme** [comment]
@darshankawar I see it also without platform view. Added a comment to https://github.com/flutter/flutter/issues/119322

**Comment by darshankawar** [comment]
Thanks for the update @delfme. Does that mean we can track this issue in the linked one and close this ?

**Comment by delfme** [comment]
@darshankawar what if we edit the title and keep this specifically for flutter android? I mean 119322 seems focusing only on flutter web.

Can you reproduce this scrolling issue on your side with latest stable/master? You could test with a simple list of ColoredBox widgets containing a text element

**Comment by darshankawar** [comment]
@delfme 
We tried to replicate this behavior using the code sample you provided here, but we couldn't consistently see the jank, apart from occasional jank.
Can you enable `performanceOverlay` and share a video of it that shows the consistent jank / performance overlay ?
Also would be helpful to get the device you are seeing this on, refresh rate of it.

Please run the code sample in `--profile` mode and enable trace (`flutter run --profile --trace-skia`) and if you could share the trace file generated would be helpful to debug further.

See [this](https://github.com/flutter/flutter/issues/74833#issue-795394794) for further details.

**Comment by darshankawar** [comment]
Closing this in favor of (https://github.com/flutter/flutter/issues/120205) and reopening it.
